internal/bundle/generate_job_test.go

```diff
@@ -55,7 +55,7 @@ func TestAccGenerateFromExistingJobAndDeploy(t *testing.T) {
 	require.NoError(t, err)
 	generatedYaml := string(data)
 	require.Contains(t, generatedYaml, "notebook_task:")
-	require.Contains(t, generatedYaml, "notebook_path: ../src/test.py")
+	require.Contains(t, generatedYaml, fmt.Sprintf("notebook_path: %s", filepath.Join("..", "src", "test.py")))
```

Comment:
@andrewnester Why not use posix paths on Windows as well? All of the translation logic will do `FromSlash` to convert it to the right separator when doing local file lookups.

Related question: do we use the right separator when on Windows and we reference absolute workspace paths?